tests/validation/tests/v3_api/resource/terraform/k3s/master/install_k3s_master.sh

```diff
@@ -45,7 +45,7 @@ then
   systemctl restart systemd-sysctl
   cat /tmp/audit.yaml > /var/lib/rancher/k3s/server/audit.yaml
 
-  if [[ "${4}" == *"v1.18"* ]] || [[ "${4}" == *"v1.19"* ]] || [[ "${4}" == *"v1.20"* ]]
+  if [[ "${4}" == *"v1.18"* ]] || [[ "${4}" == *"v1.19"* ]] || [[ "${4}" == *"v1.20"* ]] || [[ "${4}" == *"v1.21"* ]] || [[ "${4}" == *"v1.22"* ]]
```

Comment:
Question: could you explain why?

Reply:
I've reverted this change because there was some confusion, with some mistaking it for a Go version.

Thanks for highlighting this.